server/testutils/factories/cas1NonArrival.ts

```diff
@@ -0,0 +1,12 @@
+import { Factory } from 'fishery'
+import { Cas1NonArrival } from '@approved-premises/api'
+import { faker } from '@faker-js/faker'
+import { DateFormats } from '../../utils/dateUtils'
+
+export default Factory.define<Cas1NonArrival>(() => ({
+  reason: faker.word.words(3),
+  notes: faker.lorem.words(50),
+  confirmedAt: DateFormats.dateObjToIsoDateTime(
+    new Date(faker.date.recent().getTime() + faker.number.int({ max: 1000 * 60 * 60 * 24 })),
```

Comment:
Still don't quite get why this manipulation is needed -- what's wrong with simply using `faker.date.recent()`, which does add time info?